queries.yaml

```diff
@@ -152,7 +152,7 @@ layers:
   water:
     geometry_types: [Point, MultiPoint, Polygon, MultiPolygon, LineString, MultiLineString]
     simplify_before_intersect: true
-    simplify_start: 8
+    simplify_start: 0
```

Comment:
All the transforms already run on Natural Earth and OSM features, the `simplify_start` is independent of those and evaluated on each feature on the layer.